test/test_rospy/test/unit/test_rospy_msg.py

```diff
@@ -224,7 +225,6 @@ def validate_vals(vals, teststrs=teststrs):
         #deserialize multiple values with max_msgs
         max_msgs = 5
         b.truncate(0)
-        b.seek(0)
```

Comment:
This one is not necessary since line 221 already ensures that the read position is zero.